src/Data/Primitive/Extended.hs

```diff
@@ -0,0 +1,21 @@
+-- | "Data.Primitive" extended with extra definitions.
+--
+-- Based on the ".Extended Modules" pattern:
+-- https://jaspervdj.be/posts/2015-01-20-haskell-design-patterns-extended-modules.html
```

Comment:
Thanks for the link! I was not aware that this pattern had a name, good to know! :writing_hand: